README.md

```diff
@@ -93,4 +108,13 @@ $ npm run rebuild
 - If electron shows the error: "A DLL initialization routine has failed." rebuild the electron dependencies.
 ```bash
 $ npm run rebuild
+```
+
+- If you have trouble installing dependencies, remove all the lockfiles and try again.
+
+```bash
+$ rm -rf app/yarn.lock
+$ rm -rf app/package-lock.json
+$ rm -rf yarn.lock
+$ rm -rf package-lock.json
```

Comment:
Funny and sad that we have to say to this... Maybe we should put it in a `npm run rm-lockfiles` script?